Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_001.HttpAuthorizationRequestHeader.sarif

```diff
@@ -60,14 +60,14 @@
                   "uriBaseId": "SRC_ROOT"
                 },
                 "region": {
-                  "startLine": 2,
-                  "startColumn": 30,
+                  "startLine": 4,
+                  "startColumn": 43,
                   "endLine": 4,
                   "endColumn": 70,
-                  "charOffset": 51,
-                  "charLength": 137,
+                  "charOffset": 161,
+                  "charLength": 27,
                   "snippet": {
-                    "text": "https://example.com\");\r\nvar request = new RestRequest(Method.GET);\r\nrequest.AddHeader(\"Authorization\", \"Basic SomeAuthorizationKey1111111"
+                    "text": "SomeAuthorizationKey1111111"
```

Comment:
>SomeAuthorizationKey1111111 [](http://example.com/codeflow?start=29&length=27)

Now this plug in generally sets the selection region to the secret itself. This has implications for validation, as you know, this region is insufficient to revalidate the finding, except for some classes of token/key/pat.